tests/zfs-tests/tests/functional/inuse/inuse.cfg

```diff
@@ -135,4 +135,4 @@ export PIDUFSRESTORE=""
 export BLOCK_SIZE=$(( 1024 * 1024 ))
 # number of blocks to write == size of file
 export BLOCK_COUNT=100
-export STF_TIMEOUT=1200			# 20 minutes max.
+export STF_TIMEOUT=3600			# 60 minutes max.
```

Comment:
Considering tests frequently already take more than 5 hours anyway, this shouldn't pose many practical problems.
Although maintainers might prefer not to "sneak this" into this PR.